pallets/subtensor/src/macros/events.rs

```diff
@@ -13,6 +14,14 @@ mod events {
         NetworkAdded(NetUid, u16),
         /// a network is removed.
         NetworkRemoved(NetUid),
+        /// a subnet's deregistration queue entry was scheduled.
+        SubnetDeregistrationPriorityScheduled(NetUid, BlockNumberFor<T>),
```

Comment:
```suggestion
        SubnetDeregistrationPriorityScheduled(NetUid, BlockNumberForEnqueueing<T>),
```
I'd be more explicit on the arg name as you cannot quickly infer from the context what this block number is for exactly.

Reply:
That's a type, not an argument name